### ISSUE TITLE
Help - How could I output sass map file?

### ISSUE THREAD

**Issue by KouWakai** [gha-utilities/sass-build#200]
I would like to create sass map file when the sass compiled.

Is there any option to output the file?

**Comment by S0AndS0** [comment]
**TLDR**

I believe most, if not all, Node Sass command-line options _should_ be supported within the `with` configuration block.

For example to call `--source-map`, define `source_map: true` within the YAML Workflow file;

```YAML
# ...
      - name: Compile CSS from SCSS files
        uses: gha-utilities/sass-build@v0.1.9
        with:
          source: ./_scss/main.scss
          destination: ./assets/css/main.css
          source_map: true
          source_map_embed: true
# ...
```

---

**More story**

The official [GitHub `sass/node-sass`](https://github.com/sass/node-sass) documentation mentions the following relevant command line options;

```
    --source-map               Emit source map
    --source-map-contents      Embed include contents in map
    --source-map-embed         Embed sourceMappingUrl as data URI
    --source-map-root          Base path, will be emitted in source-map as is
```

... And glancing through the source code of this repository reveals each _should_ be parsed from GitHub Actions environment variables;

```javascript
const boolean_gha_input_names = [
  /* ... */
  'omitSourceMapUrl',   // Boolean, when `true` compiled CSS is not linked to source map
  'sourceMapContents',  // Boolean, when `true` embeds contents of Sass files that contributed to compiled CSS
  'sourceMapEmbed',     // Boolean, when `true` embeds source map within compiled CSS
];
/* ... */
const string_gha_input_names = [
  /* ... */
  'sourceMapRoot',      // String, prepended to all links from from source map to Sass files
]

  /* ... */
  render_options['sourceMap'] = sourceMap;
  /* ... */
```

**Relevant source code assignments**

- https://github.com/gha-utilities/sass-build/blob/baf205e8175cf9f32a949ec76051f655a2c0eb60/index.js#L42-L44
- https://github.com/gha-utilities/sass-build/blob/baf205e8175cf9f32a949ec76051f655a2c0eb60/index.js#L59
- https://github.com/gha-utilities/sass-build/blob/baf205e8175cf9f32a949ec76051f655a2c0eb60/index.js#L101-L107

**Comment by KouWakai** [comment]
@S0AndS0 
Thank you for the detailed explanation.

I would use `source_map: true` for mine.
